vm/systemSmartContracts/esdt.go

```diff
@@ -631,6 +688,25 @@ func (e *esdt) wipe(args *vmcommon.ContractCallInput) vmcommon.ReturnCode {
 	return vmcommon.Ok
 }
 
+func (e *esdt) wipeSingleNFT(args *vmcommon.ContractCallInput) vmcommon.ReturnCode {
+	if len(args.Arguments) != 3 {
+		e.eei.AddReturnMessage("invalid number of arguments, wanted 3")
+		return vmcommon.FunctionWrongSignature
+	}
+	token, returnCode := e.basicOwnershipChecks(args)
+	if returnCode != vmcommon.Ok {
+		return returnCode
+	}
+	if string(token.TokenType) == core.FungibleESDT {
+		e.eei.AddReturnMessage("only non fungible tokens can be wiped per nonce")
+		return vmcommon.UserError
+	}
+
+	composedArg := append(args.Arguments[0], args.Arguments[1]...)
```

Comment:
can you please add above the function's signature the expected arguments?

Reply:
Will add in next PR.